build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -x
+
+if [[ "$1" == "" ]]; then
+    BUILD_DIR="${HOME}/.local/share/coredns-machine-kubernetes-build"
+    if [[ ! -d "${PWD}/build" ]]; then
+        mkdir -p "$BUILD_DIR"
+    fi
+    mkdir -p "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes"
+    cp -r . "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes"
+    chcon -Rv -t container_file_t "$BUILD_DIR"
+    sudo podman run --rm \
+        -v "${BUILD_DIR}:/go" \
+        registry.svc.ci.openshift.org/openshift/release:golang-1.11 \
+        /go/src/github.com/metalkube/coredns-machine-kubernetes/build.sh $(id -u)
+    if [[ -f "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes/coredns" ]]; then
+        cp "${BUILD_DIR}/src/github.com/metalkube/coredns-machine-kubernetes/coredns" .
+    fi
+else
+    echo "Building CoreDNS with machinekubernetes..."
+
+    go get -v github.com/coredns/coredns
+
+    pushd /go/src/github.com/coredns/coredns
```

Comment:
Shouldn't this (and any reference to /go) be $GOPATH? For building outside of a container?

Reply:
in that base container image GOPATH is /go